Feature/snowflake tutorial ecc 4683

rpietzsch opened the pull request.

seebi commented on docs/build/snowflake-tutorial/22.2-Product-name-database.png:
All snowflake screenshots should be prefixed with `snowlflake-` not `22.2.`.

All files should be lower cased only, (not `Product`).

seebi commented on docs/build/snowflake-tutorial/22.2-add-databasenew.png:
Why do we have 2 screens of the same content? Please remove one.

seebi commented on docs/build/snowflake-tutorial/22.2-add-projectnew.png:
same file again?

seebi commented on docs/build/snowflake-tutorial/22.2-add-schema.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-add-value.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-advanced.png:
cropping not correct

naming to generic

seebi commented on docs/build/snowflake-tutorial/22.2-create-project.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-custome-pattern.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-databas-name.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-database-created.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-easynav-graph.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-graph-target.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-graph-uri.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-graph-urinew.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-graph.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-jdbc-created.-new.png:
cropping not correct

file name wrong

remove all "new" from names

seebi commented on docs/build/snowflake-tutorial/22.2-jdbc-created.-newpng:
file name wrong

seebi commented on docs/build/snowflake-tutorial/22.2-jdbc-created.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-mapping-rule.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-mapping-rulenew.png:
remove all `new` from names

seebi commented on docs/build/snowflake-tutorial/22.2-organization-number.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-output.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-project-name.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-suggestion-result.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-suggestion-resultnew.png:
remove all `new` from names

seebi commented on docs/build/snowflake-tutorial/22.2-suggestions.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-suggestionsnew.png:
remove all `new` from names

seebi commented on docs/build/snowflake-tutorial/22.2-target-entity-newpng.png:
remove all `new` from names

seebi commented on docs/build/snowflake-tutorial/22.2-target-product.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-target-property.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-target-propertynew.png:
remove all `new` from names

seebi commented on docs/build/snowflake-tutorial/22.2-tick.png:
cropping not correct

seebi commented on docs/build/snowflake-tutorial/22.2-ticknew.png:
remove all `new` from names

seebi reviewed (changes requested):
Please remove all old files
Please rename files accordingly

I can do a second review afterwards.

rpietzsch commented:
@seebi please recheck.

- deleted duplicate / obsolete images
- images using style class (`bordered`) instead of of inline markup
- changed image filename prefix to `snowflake-`
- refined cropping of some images (removed borders, scrollbars, etc.)
